magefile.go

```diff
@@ -2,6 +2,7 @@
 // or more contributor license agreements. Licensed under the Elastic License;
 // you may not use this file except in compliance with the Elastic License.
 
+//go:build mage
 // +build mage
```

Comment:
nit: are both comments needed?

Reply:
I think this was added by my local go format (1.17), I will remove the old one when applying #2433 to staging.